pkg/v1/builder/template/plugintemplates/Makefile.tmpl

```diff
@@ -11,6 +11,8 @@ GOARCH ?= $(shell go env GOARCH)
 GOHOSTOS ?= $(shell go env GOHOSTOS)
 GOHOSTARCH ?= $(shell go env GOHOSTARCH)
 
+PLUGIN_PATH ?= ./cmd/cli/plugin
```

Comment:
I think this should be `./cmd/plugin` for the generated plugins

Reply:
PLUGIN_PATH represents where it needs to search for plugin files.

Reply:
After our discuss, i agree that it should be ./cmd/plugin

Reply:
changes done.